### PR TITLE
Fixes #8293

### LINKED ISSUE #8293: Cuban pete arcade easily cheesed
Before to cheat you had to unscrew it and screw it back. Now all you have to do is emag it again and it resets. This means bombcap bombs can be easily gotten with only 3 tc (cost of an emag)

### PR DESCRIPTION
**Pull request by ghost**
Fixes #8293
Also removes some trailing returns.

### CODE REVIEW

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -185,10 +185,11 @@
 		if(emagged)
 			src.New()
 			emagged = 0
+			emagger = null
+			emagger.locked_to = null
```
Comment (PJB3005) [inline code/game/machinery/computer/arcade.dm]: What

File: code/game/machinery/computer/arcade.dm [hunk 2]
```diff
@@ -263,19 +264,22 @@
...
+		return
 	temp = "If you die in the game, you die for real!"
+	to_chat(user,"<span class='danger'>Your legs become buckled to the machine, and you are unable to move unless you defeat Cuban Pete!")
 	player_hp = 30
 	player_mp = 10
 	enemy_hp = 45
 	enemy_mp = 20
 	gameover = 0
 	blocked = 0
+	user.locked_to = src
```
Comment (unid15) [inline code/game/machinery/computer/arcade.dm]: src.lock_atom(user)

Comment (ghost) [inline code/game/machinery/computer/arcade.dm]: How do I unlock them, then?

File: code/game/machinery/computer/arcade.dm (hunk 1 above)
Comment (ghost) [inline code/game/machinery/computer/arcade.dm]: They're locked to the machine now.

File: code/game/machinery/computer/arcade.dm (hunk 2 above)
Comment (unid15) [inline code/game/machinery/computer/arcade.dm]: src.unlock_atom(user)

**Comment by CptWad** [comment]
Well I guess thats one way to do it
Prepare for tears

**Comment by ihadtoregisterforthis** [comment]
THE RIDE NEVER ENDS

**Comment by clusterfack** [comment]
I'm getting a 'I didn't test this vibe' for some reason, make sure it works with locking/unlocking all that

**Comment by 9600bauds** [comment]
This is extremely lame because it doesn't allow you to let OTHER people play unless you give them your emag. My earliest encounter with Outbomb Cuban Pete was as a clueless chef that didn't know it gibbed you, a tator mechanic had built a ton of arcades at the bar and emagged them all. Honestly, it was funny as fuck.
Make it lock you when you start playing and maybe

**Comment by 9600bauds** [comment]
This also needs to null emagger on Destroy()

**Comment by PJB3005** [comment]
In fact this gonna need to properly listen to the emagger's `on_destroyed` to dereference `emagger` when the emagger gets deleted.

**Comment by Boxe** [comment]
c h a n g e l o g

**Comment by Probe1** [comment]
CELT

**Comment by 9600bauds** [comment]
Nunca Jamás

**Comment by Probe1** [comment]
el código del muerto 

:skull:

**Pull request closed without merging**